docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,178 @@ Identify the client to the server and negotiate the protocol version.
+
+**Example Results**::
+    
+  When nothing is sent all masternodes information is received.
+  
+    [
+      {
+        "vin": "9d298c00dae8b491d6801f50cab2e0037852cb556c5619ddb07c50421x9a31ab",
+        "status": "ENABLED",
+        "protocol": 70213,
+        "payee": "PDFHmjKLvSGdnWgDJSJX49Rrh0SJtRANcE",
+        "lastseen": "2018-04-01 12:34",
+        "activeseconds": 1258000,
+        "lastpaidtime": "2018-03-15 05:29",
+        "lastpaidblock": 1234,
+        "ip": "1.0.0.1",
+        "paymentposition": 3333,
+        "balance": 510350
+      },
+      {
+        "vin": "9d298c00dae8b491d6801f50cab2e0037852cb556c5619ddb07c50421x9a31ac",
+        "status": "ENABLED",
+        "protocol": 70213,
+        "payee": "PDFHmjKLvSGdnWgDJSJX49Rrh0SJtRANcE",
+        "lastseen": "2018-04-01 12:34",
+        "activeseconds": 1258000,
+        "lastpaidtime": "2018-03-15 05:29",
+        "lastpaidblock": 1234,
+        "ip": "1.0.0.2",
+        "paymentposition": 3333,
+        "balance": 520700
+      },
+      ...,
+      ...,
+      ...,
+      ...
+    ]
+  
+  When a single string is sent, only one record is received.
+  
+    {
+      "vin": "9d298c00dae8b491d6801f50cab2e0037852cb556c5619ddb07c50421x9a31ab",
+      "status": "ENABLED",
+      "protocol": 70213,
+      "payee": "PDFHmjKLvSGdnWgDJSJX49Rrh0SJtRANcE",
+      "lastseen": "2018-04-01 12:34",
+      "activeseconds": 1258000,
+      "lastpaidtime": "2018-03-15 05:29",
+      "lastpaidblock": 1234,
+      "ip": "1.0.0.1",
+      "paymentposition": 3333,
+      "balance": 510350
+    }
+  
+  When an array is sent, an array is received.
+  
+    [
+      {
+        "vin": "9d298c00dae8b491d6801f50cab2e0037852cb556c5619ddb07c50421x9a31ab",
+        "status": "ENABLED",
+        "protocol": 70213,
+        "payee": "PDFHmjKLvSGdnWgDJSJX49Rrh0SJtRANcE",
+        "lastseen": "2018-04-01 12:34",
+        "activeseconds": 1258000,
+        "lastpaidtime": "2018-03-15 05:29",
+        "lastpaidblock": 1234,
+        "ip": "1.0.0.1",
+        "paymentposition": 3333,
+        "balance": 510350
+      },
+      {
+        "vin": "9d298c00dae8b491d6801f50cab2e0037852cb556c5619ddb07c50421x9a31ac",
+        "status": "ENABLED",
+        "protocol": 70213,
+        "payee": "PDFHmjKLvSGdnWgDJSJX49Rrh0SJtRANcE",
+        "lastseen": "2018-04-01 12:34",
+        "activeseconds": 1258000,
+        "lastpaidtime": "2018-03-15 05:29",
+        "lastpaidblock": 1234,
+        "ip": "1.0.0.2",
+        "paymentposition": 3333,
+        "balance": 520700
+      }
+    ]
+
+masternode.info
+--------------
+
+Returns the full info of masternode.
+
+**Signature**
+
+  .. function:: masternode.info(masternode_payee)
+  
+  * *masternode_payee*
+
+    Masternode payee address.
+
+**Result**
+
+  An array with the full information of the masternode.
+  
+**Examples**::
+
+  .. masternode.info("9d298c00dae8b491d6801f50cab2e0037852cb556c5619ddb07c50421x9a31ab")
```

Comment:
This doesn't look like a Dash address.  Is it a public key?  Or a TX ID?  The documentation of each method should be clear about all the inputs.

Reply:
sorry my bad, it should be an address.